src/Game.js

```diff
@@ -4,6 +4,9 @@ import ColorShape from './ColorShape';
 import ColorUtil from './util/ColorUtil';
 import ColorQueue from './ColorQueue';
 
+const SELECT_THICKNESS = 6;
+const CURRENT_THICKNESS = 8;
```

Comment:
CURRENT THICKNESS is 8 but default is 4? What does current thickness mean? Consider renaming.